.github/workflows/test-changes-exists-drift.yml

```diff
@@ -50,7 +54,8 @@ jobs:
           sha: ${{ github.sha }}
           drift-detection-mode-enabled: true
           atmos-config-path: ${{ runner.temp }}
-          atmos-version: 1.81.0
+          atmos-version: 1.92.0
+          do-not-checkout: true
```

Comment:
```suggestion
          checkout-enabled: false
```

Reply:
We like to keep things positive. =)  So, rather than negative a negative (e.g. a double negative), assume the positive (checkout-enabled) and negate that.